cafebabel/static/css/styles.css

```diff
@@ -2029,17 +2030,25 @@ article .article-detail p span {
   content: '#';
 }
 
-.tags-list a.impact,
-.tags-list a.experience,
-.tags-list a.raw,
-.tags-list a.creative {
+.article-page article p.tags-list a.impact,
```

Comment:
Is `.article-page .tags-list a.impact` not specific enough?

Reply:
no :-( cause it gets overridden by `.article-page article p.tags-list a, .tags-list a` line 2004